.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: "build-test"
+on:
+  push:
+    branches:
+      - net203
+  pull_request:
+    branches:
+      - net203
+
+jobs:
+  test-backend:
+    runs-on: ${{ matrix.image }}
+    strategy:
+      matrix:
+        image: [windows-latest]
+    env:
+      NUGET_PACKAGES: ${{ github.workspace }}/.github/nuget-packages
+      GRADLE_USER_HOME: ${{ github.workspace }}/.github/gradle
+    steps:
+      - uses: actions/checkout@v2
+
+      # Prepare
+      - name: Gradle JVM Cache
+        uses: actions/cache@v1.1.0
+        with:
+          path: build/gradle-jvm
+          key: ${{ runner.os }}.gradle-jvm.${{ hashFiles('rider-fsharp/gradle*') }}
+
+      - name: Gradle Wrapper Cache
+        uses: actions/cache@v1.1.0
+        with:
+          path: ${{ env.GRADLE_USER_HOME }}/wrapper
+          key: ${{ runner.os }}.gradle-wrapper.${{ hashFiles('rider-fsharp/gradle/**') }}
+
+      - name: NuGet Cache
+        uses: actions/cache@v1.1.0
+        with:
+          path: ${{ env.NUGET_PACKAGES }}
+          key: ${{ runner.os }}.nuget.${{ hashFiles('ReSharper.FSharp/**/*.csproj') }}+${{ hashFiles('ReSharper.FSharp/**/*.props') }}
```

Comment:
Is there a reason you're only interested in `csproj` projects and not in `fsproj` ones?

Reply:
oh, really